.github/workflows/release.yaml

```diff
@@ -24,7 +37,6 @@ jobs:
       - run: echo "tag=$(jq .version package.json -r)" >> $GITHUB_OUTPUT
         name: Print future release version
         id: version_bump
-      - run: gh release create ${{ steps.version_bump.outputs.tag }} --generate-notes
-        env:
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        name: Create the release
+      - run: git commit -m 'build ${{ steps.version_bump.outputs.tag }}'
+      - run: git tag ${{ steps.version_bump.outputs.tag }}
```

Comment:
Tag will be an auto-increment of current minor package version. Next tag will be 2.2.0